src/app/component/eco-news/eco-news-routing.module.ts

```diff
@@ -0,0 +1,38 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { EcoNewsComponent } from './eco-news.component';
+import { CreateNewsComponent } from './create-news/create-news.component';
+import { EcoNewsDetailComponent } from './eco-news-detail/eco-news-detail.component';
+import {NewsPreviewPageComponent} from './news-preview-page/news-preview-page.component';
+import {PostNewsLoaderComponent} from './post-news-loader/post-news-loader.component';
```

Comment:
Please in this line and in the line above add two spaces between module name

Reply:
fix